modules/understanding-dns-operator.adoc

```diff
@@ -0,0 +1,21 @@
+// Module filename: nw-creating-secured-routes.adoc
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+[id="understanding-dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator deploys and manages CoreDNS to provide a name resolution
+service to pods that enables DNS-based Kubernetes Service discovery in
+OpenShift. It manages CoreDNS as a Kubernetes DaemonSet exposed as
+a Service with a static IP, and CoreDNS runs on all nodes in the cluster.
+
+The DNS Operator is deployed during installation and creates a working default
+deployment based on the cluster's configuration.
+
+The default cluster domain is `cluster.local`.
+
```

Comment:
Since you mention the default domain above, you may also want to mention the default Service IP used for DNS. This is `.10` from the service cidr. For example:
```
# To find the service cidr
$ oc get networks.config/cluster -o jsonpath='{$.status.serviceNetwork}'
[172.30.0.0/16]

# To find the service ip used for dns:
$ oc get service/dns-default -n openshift-dns -o jsonpath='{$.spec.clusterIP}'
172.30.0.10
```